### PR TITLE
added contains() method to Cover class.

### PR DESCRIPTION
**Pull request by pulquero**
See #208

### CODE REVIEW

File: test/test_coverer.py [hunk 1]
```diff
@@ -76,6 +76,14 @@ def test_perc_overlap(self, CoverClass):
         for i in (1, 2, 3, 4):
             assert data[i] in entries[1]
 
+    def test_contains(self, CoverClass):
```
Comment (sauln) [inline test/test_coverer.py:79]: could you add a test that shows this works in higher number of dimensions as well?

**Comment by deargle** [comment]
Low-effort comment warning: the semantics of `contains()` suggests to me that it would accept a datapoint and return a boolean if any of the nodes within any of the covers' hypercubes contains that exact node. But of course that doesn't make sense, because covers don't know about points that might be within its hypercubes. And of course, knowing which hypercube contains a datapoint doesn't on its own, reach the ultimate goal of knowing which _node_ a point is closest to. Is the intention that, once the hypercube is known any nodes/clusters within those hypercubes count as "closest"? Hmm.

**Comment by pulquero** [comment]
Should I rename it to find_cubes()? Note, this PR is an incremental step towards the goal, rather than having one PR with lots of changes to review.

**Comment by deargle** [comment]
I like `find_cubes()`, @sauln?

:+1: for it being incremental. Thinking ahead, what's the algorithm? 

* For a datapoint, find hypercubes
* for each hypercube, rerun the clustering algorithm, including the new point?

**Comment by pulquero** [comment]
For each cube, find the nearest neighbors, then lookup the clusters of the nearest neighbors (code in #208). I'm not trying to do incremental/updateable tda, but rather using the tda result to cheaply tell me something about new data.

**Comment by pulquero** [comment]
Or maybe find_sets (~ open set) to be shape agnostic.

**Comment by deargle** [comment]
> Or maybe find_sets (~ open set) to be shape agnostic.

Reviewing the lit, yes, `find_open_sets` would be more generic. There's several other references to "cubes" in the current cover class which could be changed later. Unless we argue that covers _only_ ever contain open sets, in which case, just `find()` would suffice.

**Comment by sauln** [comment]
Looks pretty good to me!  Thanks for the contribution :D

**Pull request merged**